CONTRIBUTING.md

```diff
@@ -132,10 +132,10 @@ The `internal` subpackages contain most of the FerretDB code:
 - `handlers/sjson` provides converters from/to SJSON for built-in and `types` types.
   SJSON adds some extensions to JSON for keeping object keys in order,
   preserving BSON type information in the values themselves, etc.
-  It is used by `sqlite` and `pg` handlers.
+  It is used by `sqlite` and old `pg` handlers.
```

Comment:
We use it in new `postgresql` backend too, for mashal and stuff. Is that a problem?